prow/github/client.go

```diff
@@ -651,3 +652,41 @@ func (c *Client) FindIssues(query string) ([]Issue, error) {
 	}, &issSearchResult)
 	return issSearchResult.Issues, err
 }
+
+// set the trusted orgs
```

Comment:
See effective go for doc comment conventions: https://golang.org/doc/effective_go.html#commentary

In particular, this should start with "SetTrustedOrgs" and be a complete sentence.